src/components/MultiSelectCombobox/UncontrolledMultiSelectCombobox.tsx

```diff
@@ -30,22 +37,39 @@ function isOptionFiltered<T extends ComboboxOption>(option: T, filterText: strin
 }
 
 const defaultProps = {
+  components: {},
```

Comment:
nit: I don't think you need an empty object as a default prop. Object spread is fine for `undefined`:
```js
{ ...undefined } // {}
```